fakes/calculator.go

```diff
@@ -0,0 +1,29 @@
+package fakes
```

Comment:
question: just to confirm everything in `fakes` is automatically generated. And this generation does not need to be triggered manually - go will take care of it, right? I think I saw the build flag comments somewhere.

Reply:
no it's not generated, just mock implementations of paketo stuff. Unless it's generated code in the paketo repos but I didn't see anything that would generate it.